dashboards-observability/public/components/visualizations/charts/coordinate_map/coordinate_map.tsx

```diff
@@ -16,23 +17,30 @@ import { visChartTypes } from '../../../../../common/constants/shared';
 export const CoordinateMap = ({ visualizations }: any) => {
   const { explorerData } = useContext<any>(TabContext);
   const { dataConfig = {}, layoutConfig = {} } = visualizations?.data?.userConfigs;
-  const dataConfiguration = visualizations?.data?.rawVizData?.scattergeo?.dataConfig;
   const rawData = explorerData.jsonData;
 
+  if (isEmpty(rawData)) {
+    return <EmptyPlaceholder icon={visualizations?.vis?.icontype} />;
+  }
+
   const fetchPlotNames = () =>
-    rawData.map((data: any) => data?.[dataConfiguration?.metrics[0]?.plotName]);
+    rawData.map((data: any) => data?.[dataConfig?.valueOptions?.metrics[0]?.plotName]);
```

Comment:
Could we add types for these any? we may need to gradually replace any with specific types

Reply:
@mengweieric , this is the data coming from the API response. That is why the type is given as any. Do you want me to give the type as Object or something?